score/ipython/scoring_comparison.ipynb

```diff
@@ -27,375 +46,834 @@
-    "# Calculate the top K% of prioritized communities\n",
-    "cejst_df[CEJST_PRIORITY_COMMUNITY_FIELD] = (\n",
-    "    cejst_df[CEJST_PERCENTILE_FIELD] >= CEJST_PRIORITY_COMMUNITY_THRESHOLD\n",
-    ")\n",
+    "# # Rename unclear name \"id\" to \"census_block_group_id\", as well as other renamings.\n",
+    "# cejst_df.rename(\n",
+    "#     columns={\n",
+    "#         \"Total population\": CENSUS_BLOCK_GROUP_POPULATION_FIELD,\n",
+    "#         score_used: CEJST_SCORE_FIELD,\n",
+    "#         f\"{score_used} (percentile)\": CEJST_PERCENTILE_FIELD,\n",
+    "#     },\n",
+    "#     inplace=True,\n",
+    "#     errors=\"raise\",\n",
+    "# )\n",
     "\n",
     "# Create the CBG's Census Tract ID by dropping the last number from the FIPS CODE of the CBG.\n",
     "# The CBG ID is the last one character.\n",
     "# For more information, see https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html.\n",
-    "cejst_df.loc[:, CENSUS_TRACT_ID_FIELD] = (\n",
-    "    cejst_df.loc[:, CENSUS_BLOCK_GROUP_ID_FIELD].astype(str).str[:-1].astype(np.int64)\n",
+    "cejst_df.loc[:, GEOID_TRACT_FIELD_NAME] = (\n",
+    "    cejst_df.loc[:, GEOID_FIELD_NAME].astype(str).str[:-1]\n",
     ")\n",
     "\n",
-    "# Remove all non-California data\n",
-    "cejst_df = cejst_df.loc[\n",
-    "    cejst_df[CENSUS_BLOCK_GROUP_ID_FIELD].astype(str).str[0] == \"6\", :\n",
-    "]\n",
+    "cejst_df.loc[:, GEOID_STATE_FIELD_NAME] = (\n",
+    "    cejst_df.loc[:, GEOID_FIELD_NAME].astype(str).str[0:2]\n",
+    ")\n",
     "\n",
     "cejst_df.head()"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ec6b27e3",
-   "metadata": {},
+   "id": "08962382",
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
-    "# Data from https://calenviroscreen-oehha.hub.arcgis.com/#Data, specifically:\n",
-    "# https://oehha.ca.gov/media/downloads/calenviroscreen/document/calenviroscreen40resultsdatadictionaryd12021.zip\n",
+    "# Load CalEnviroScreen 4.0\n",
+    "CALENVIROSCREEN_SCORE_FIELD = \"calenviroscreen_score\"\n",
+    "CALENVIROSCREEN_PERCENTILE_FIELD = \"calenviroscreen_percentile\"\n",
+    "CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD = \"calenviroscreen_priority_community\"\n",
     "\n",
-    "download = requests.get(\n",
-    "    \"https://justice40-data.s3.amazonaws.com/CalEnviroScreen/CalEnviroScreen_4.0_2021.zip\",\n",
-    "    verify=False,\n",
+    "calenviroscreen_data_path = DATA_DIR / \"dataset\" / \"calenviroscreen4\" / \"data06.csv\"\n",
+    "calenviroscreen_df = pd.read_csv(\n",
+    "    calenviroscreen_data_path, dtype={GEOID_TRACT_FIELD_NAME: \"string\"}\n",
     ")\n",
-    "file_contents = download.content\n",
-    "zip_file_path = TEMP_DATA_DIR\n",
-    "zip_file = open(zip_file_path / \"downloaded.zip\", \"wb\")\n",
-    "zip_file.write(file_contents)\n",
-    "zip_file.close()"
+    "\n",
+    "# Convert priority community field to a bool.\n",
```

Comment:
It's inconsistent that it seems we're doing this conversion here in the scoring comparison notebook for calenviroscreen but we do it in the HUD RECAP ETL for the relevant field in the recap data.